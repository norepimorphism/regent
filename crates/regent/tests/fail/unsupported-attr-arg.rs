fn main() {
    // Unsupported.
    #[regent::bitwise(foo = 0)]
    struct A;
    // Supported, unsupported.
    #[regent::bitwise(size = 1, foo = 0)]
    struct B;
    // Unsupported, supported.
    #[regent::bitwise(foo = 0, size = 1)]
    struct C;
    // Unsupported, unsupported.
    #[regent::bitwise(foo = 0, bar = 0)]
    struct D;
}
