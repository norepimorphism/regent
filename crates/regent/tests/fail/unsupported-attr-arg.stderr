error: argument is not supported
 --> tests/fail/unsupported-attr-arg.rs:3:23
  |
3 |     #[regent::bitwise(foo = 0)]
  |                       ^^^

error: argument is not supported
 --> tests/fail/unsupported-attr-arg.rs:6:33
  |
6 |     #[regent::bitwise(size = 1, foo = 0)]
  |                                 ^^^

error: argument is not supported
 --> tests/fail/unsupported-attr-arg.rs:9:23
  |
9 |     #[regent::bitwise(foo = 0, size = 1)]
  |                       ^^^

error: argument is not supported
  --> tests/fail/unsupported-attr-arg.rs:12:23
   |
12 |     #[regent::bitwise(foo = 0, bar = 0)]
   |                       ^^^
